build.sbt

```diff
@@ -0,0 +1,42 @@
+import sbt._
+
+enablePlugins(ScalazPlugin)
```

Comment:
@vil1 One quick question. If we remove `enablePlugins(ScalazPlugin)` from build.sbt, how project identify about `ScalazPlugin`?

Reply:
The plugin is "brought in" by the `dependsOn(RootProject(...))` line in `project/plugins.sbt`.
It is automatically activated because of [this line](https://github.com/scalaz/scalaz-sbt/blob/730be174fe4812babbed10e9cdfb3efcb6dd51a3/src/main/scala/ScalazPlugin.scala#L10) in the plugin source code